src/functions/connection/onSetUsername.ts

```diff
@@ -28,18 +26,14 @@ export const handler: Handler = async (event: WebSocketAPIGatewayEvent): Promise
   try {
     if (username) {
       await setUsername(connectionId, username);
-      const wsResponse: WebSocketResponse = createLoginSuccessResponse();
+      const wsResponse = createLoginSuccessResponse();
       await ws.send(wsResponse, connectionId);
-
-      return response(200, `Set username to ${username}`);
+    } else {
+      console.error('onSetUsername: username cannot be empty');
```

Comment:
Should we be sending this error back through websocket instead to let frontend know?

Reply:
this is just for the backend logs, any error occurred would be sent to the frontend via line 37 below